src/malos.cpp

```diff
@@ -106,6 +107,12 @@ int RunServer() {
   }
   driver_manager.RegisterDriver(&driver_lirc);
 
+  ServoDriver driver_servo;
+  if (!driver_servo.Init(kBasePort + 4 * 8 + 1, kUnsecureBindScope)) {
```

Comment:
After this line is where you can do:
  driver_servo.SetupWishboneBus(wishbone_bus);
(Following the comments given before).